packages/nodes-base/nodes/HttpRequest/V3/HttpRequestV3.node.ts

```diff
@@ -1152,8 +1152,8 @@ export class HttpRequestV3 implements INodeType {
 			if (timeout) {
 				requestOptions.timeout = timeout;
 			} else {
-				// set default timeout to 1 hour
-				requestOptions.timeout = 3600000;
+				// set default timeout to 5 minutes
+				requestOptions.timeout = 300_000;
```

Comment:
Will that work with the `_`? I would have expected `300000`

Reply:
Yeah it's just nice for readability, supported since Node 12 so shouldn't be an issue? https://v8.dev/features/numeric-separators

Reply:
That is neat and I have learnt something :)